redwood/api/src/services/addressInformation/addressInformation.ts

```diff
@@ -0,0 +1,18 @@
+import axios from "axios"
+import {QueryfetchAddressTransactionsArgs} from 'api/types/graphql'
+
+export const fetchAddressTransactions = async({ ethereumAddress }: QueryfetchAddressTransactionsArgs) => {
+  try {
+    const etherscanResponse = await axios.get(`https://api.etherscan.io/api?module=account&action=txlist&address=${ethereumAddress}&page=1&offset=10&apikey=${process.env.ETHERSCAN_API_KEY}`)
```

Comment:
Ok with you if we use `fetch` instead of `axios` for this? I know axios is a bit more full-featured but I find `fetch` generally works well enough, is more familiar to frontend developers and we already have it as a dependency.

Reply:
Separately, we should only attempt this query if the Etherscan API key is non-null. If it is null, there should be some reasonable fallback.

Reply:
I agree that we can assume `ethereumAddress` is non-null here. But what happens if `ETHERSCAN_API_KEY` isn't defined? In production we should probably throw an error, but in dev I think this should fail gracefully so contributors don't have to create so many accounts to get running.